packages/edit-site/src/components/dataviews/text-filter.js

```diff
@@ -10,14 +10,14 @@ import { SearchControl } from '@wordpress/components';
  */
 import useDebouncedInput from '../../utils/use-debounced-input';
 
-export default function TextFilter( { view, onChangeView } ) {
+export default function TextFilter( { id, view, onChangeView } ) {
```

Comment:
I'm not sure about making this generic. the "search" is a special filter that applies cross field, so I think it's fine to be its own thing no?

If we want to share the UI, we should just pass `value` and `onChange` instead.

Reply:
I've shared my thoughts on how I see the filters/view/fields relationship at https://github.com/WordPress/gutenberg/pull/55270/files#r1362031450